3rd-party/class.jetpack-amp-support.php

```diff
@@ -66,7 +66,7 @@ static function is_amp_request() {
 			&&
 				function_exists( 'amp_is_canonical' ) // this is really just testing if the plugin exists
 			&&
-				( amp_is_canonical() || isset( $_GET[ amp_get_slug() ] ) );
+				( amp_is_canonical() || isset( $_GET[ amp_get_slug() ] ) || self::has_amp_suffix() );
```

Comment:
The last part in this conditional should be wrapped in a check for `AMP__VERSION` being less than `1.0` since with  https://github.com/Automattic/amp-wp/issues/1148 there will no longer be an `/amp/` endpoint, and this will help guard against the case when a user publishes a page about guitar amps at like `/guitar/amp/`, to avoid ambiguity.

Reply:
👍